storage/persistence.go

```diff
@@ -59,3 +66,28 @@ func (b *Bucket) InsertLogChunks(ctx context.Context, buildID string, testID str
 
 	return nil
 }
+
+// CheckMetadata returns whether the metadata file exists for the given build
+// and (optional) test.
```

Comment:
This comment implies it checks both, when it actually checks the test metadata specifically if the testID is supplied.

Reply:
Done.